modules/quadbin/redshift/sql/QUADBIN_POLYFILL.sql

```diff
@@ -0,0 +1,42 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION @@RS_PREFIX@@carto.__QUADBIN_POLYFILL
+(geojson VARCHAR(MAX), resolution INT)
+RETURNS VARCHAR(MAX)
+STABLE
+AS $$
+    from @@RS_PREFIX@@quadbinLib import geojson_to_quadbins
+    import json
+
+    if geojson is None or resolution is None:
+        return None
+
+    if resolution < 0 or resolution > 26:
+        raise Exception('Invalid resolution, should be between 0 and 26')
+
+    pol = json.loads(geojson)
+    quadbins = []
+    if pol['type'] == 'GeometryCollection':
+        for geom in pol['geometries']:
+            quadbins += geojson_to_quadbins(
+                geom, {'min_zoom': resolution, 'max_zoom': resolution}
+            )
+        quadbins = list(set(quadbins))
+    else:
+        quadbins = geojson_to_quadbins(
+            pol, {'min_zoom': resolution, 'max_zoom': resolution}
+        )
+
+    return json.dumps(quadbins)
+$$ LANGUAGE plpythonu;
+
+CREATE OR REPLACE FUNCTION @@RS_PREFIX@@carto.QUADBIN_POLYFILL
+(GEOMETRY, INT)
+-- (geo, resolution)
+RETURNS SUPER
+STABLE
+AS $$
+    SELECT json_parse(@@RS_PREFIX@@carto.__QUADBIN_POLYFILL(ST_ASGEOJSON($1)::VARCHAR(MAX), $2))
```

Comment:
Does RS supports SRIDs in the GEOMETRY? For PG I took the trouble to support any SRID using ST_TRANSFORM.... but we could also specify that input geometries must be 4326

Reply:
Yes, RS supports different SRIDs I'll be adding `ST_SetSRID` to these kind of cases.

Reply:
Updated in latest commit